.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+on: push
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install pep517
+        run: >-
+          python -m
+          pip install
+          pep517
+          --user
+      - name: Build a binary wheel and a source tarball
+        run: >-
+          python -m
+          pep517.build
+          --source
+          --binary
+          --out-dir dist/
+          .
+      - name: Publish distribution 📦 to PyPI
+        if: startsWith(github.event.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
```

Comment:
Do we want to publish to Test Pypi too?

Reply:
Hmmm good point, I was lazy and ignored it 😆 

It makes sense to have it. But I believe in a step before, right? I mean, not in the release process. Cause then we can test it before sending to "production".

Reply:
That's correct. We'd need another workflow.

Reply:
@barbosa I created a issue for that https://github.com/scanapi/scanapi/issues/146